modules/decrypt-node/test/parse_header_stream.test.ts

```diff
@@ -39,15 +42,38 @@ describe('ParseHeaderStream', () => {
       await testStream(cmm, data.slice(0, i))
     }
   })
+
+  it('Precondition: If maxHeaderSize was set I can not buffer a header larger than maxHeaderSize.', async () => {
```

Comment:
Just to make sure, let's also add a "equal to length" test case too.

Reply:
This tests this too.  The `completeHeaderLength` holds the value, and the second for loop goes from equality, through the rest of the message.  Adding comments